packages/amplify_auth_cognito/lib/method_channel_auth_cognito.dart

```diff
@@ -320,6 +320,27 @@ class AmplifyAuthCognitoMethodChannel extends AmplifyAuthCognito {
     }
   }
 
+  @override
+  Future<Map<String, UpdateUserAttributeResult>> updateUserAttributes(
+      {required UpdateUserAttributesRequest request}) async {
+    Map<String, UpdateUserAttributeResult> res;
```

Comment:
Is this used anywhere anymore in this method?

Reply:
Oops I was only referring to line 326

Reply:
It's not thanks!